triaging.rst

```diff
@@ -3,193 +3,247 @@
+|                | improvements in the documentation, test suite and        |
+|                | other refactorings. A good place to discuss enhancements |
+|                | prior to filing an issue is `python-ideas`_ mailing      |
+|                | list.                                                    |
++----------------+----------------------------------------------------------+
+| performance    | Situations where too much time is necessary to complete  |
+|                | the task. For example, a common task now takes           |
+|                | significantly longer to complete.                        |
++----------------+----------------------------------------------------------+
+| resource usage | Situations where too many resources (e.g. memory) are    |
+|                | used.                                                    |
++----------------+----------------------------------------------------------+
+| security       | Issues that might have security implications. Report     |
+|                | security vulnerabilities using the procedure found in    |
+|                | the `Reporting security issues in Python`_ page on the   |
+|                | python.org website.                                      |
++----------------+----------------------------------------------------------+
 
 Stage
 '''''
-What is needed next to advance the issue.  The *stage* needn't be set until
-it is clear that the issue warrants fixing.
-
-test needed
-    The bug reporter should post a script or instructions to let a triager or
-    developer reproduce the issue.
-needs patch
-    The issue lacks a patch to solve the problem (i.e. fixing the bug, or
-    adding the requested improvement).
-patch review
-    There is a patch, but it needs reviewing or is in the process of being
-    reviewed. This can be done by any triager as well as a core developer.
-commit review
-    A triager performed a patch review and it looks good to them, but a core
-    developer needs to commit the patch (and do a quick once-over to make sure
-    nothing was overlooked).
-resolved
-    The issue is considered closed and dealt with.
+A needed next action to advance the issue.  The *stage* needn't be set until
+it is clear that the issue has been initially triaged and determined work
+will be needed.
+
++---------------+----------------------------------------------------------+
+|     Stage     |                       Description                        |
++===============+==========================================================+
+| test needed   | The steps which are needed to reproduce the issue. The   |
+|               | bug reporter should post a script, instructions, or      |
+|               | example to help someone test or reproduce the issue.     |
++---------------+----------------------------------------------------------+
+| needs patch   | A patch is needed to solve the problem (i.e. fixing the  |
```

Comment:
Should we use "patch or pull request" in this table, with a short note below about their functional equivalence? They're pretty much interchangeable as far as the tracker is concerned, but readers that are only familiar with GitHub may not have encountered the "patch" term.

Reply:
Great point. Will add pull request.

Reply:
Good suggestion. Will do.